openeogeotrellis/backend.py

```diff
@@ -2185,13 +2185,19 @@ def get_results_metadata(self, job_id: str, user_id: str) -> dict:
 
         return {}
 
-    def get_log_entries(self, job_id: str, user_id: str, offset: Optional[str] = None) -> Iterable[dict]:
+    def get_log_entries(
+        self,
+        job_id: str,
+        user_id: str,
+        offset: Optional[str] = None,
+        log_level: Optional[str] = None,
```

Comment:
see python driver PR:  will be `level` iso `log_level`

Reply:
Done.